### DISCUSSION TITLE
3.8.0.0rc1

### DISCUSSION THREAD

**Discussion by gsemet** [Guake/guake#1908] [Announcements]
First release candidate of Guake 3.8 after a long time without release. Just to test the release process. 

<hr /><em>This discussion was created from the release <a href='https://github.com/Guake/guake/releases/tag/'>3.8.0.0rc1</a>.</em>

**Comment by gsemet** [comment]
I prereleased this version to test release process ([pipeline here](https://github.com/Guake/guake/actions/runs/1260816248)). We may want to do several rc tags so that we have a correct publication to pypi. I'll try to find enough time and documentation to see how this can be integrated with github actions

**Comment by Davidy22** [comment]
I went and filled in the release notes on the tagged 3.8 rc1 with a modified version of the reno output. Also learned a little bit about how the reno file needs to be written from the reno output.

**Comment by gsemet** [comment]
I test the release to pypi integration in github actions with pipy-test, it does not work yet so we might have quite a lot of rc's... but that's why they are used for. i don't know github actions that much, I am more a gitlab user, but it looks pretty cool

**Reply by Davidy22** [reply]
Build's fixed and the latest version's been working for me for a good while now. Think we're good for a release, I did have to edit the release notes file a little bit when I generated the first one for the first RC because I didn't have the best grasp of how reno files would turn into release notes when I was writing them. Also, I noticed that the top voted item on feathub's a PPA, which some people have already been providing. Since we're doing a release, I was thinking we could contact them, see how enthusiastic they still are and see if they want to be featured a little more prominently as an installation path.

**Comment by gsemet** [comment]
Can you do a make release notes pull request ? I do not have access to a dev environment

**Reply by Davidy22** [reply]
Done

**Comment by gsemet** [comment]
For the ppa they dropped support some times ago but yes an official one would be cool. We have a snap, but again, someone needs to maintain it

**Reply by Davidy22** [reply]
We could still message them and see where they're at, their latest build is newer than our last release so it looks like they sustained some interest for at least a little while.

**Reply by Davidy22** [reply]
Just headed over to try message them and they literally updated two days ago, far from my definition of "dropped support." Sent the first contact, now just to see how they respond.

**Comment by Davidy22** [comment]
Been some amount of time since I updated the release notes file. Is there's anything else blocking the release @gsemet?

**Reply by gsemet** [reply]
If we are good to go, I release

**Reply by gsemet** [reply]
Can you copy paste here the Reno change log ?

we can work on changing the release procedure, if you want to be able to do it yourself. The goal is to be able to put the reno release notes in the GitHub release description.

don’t know if we can automatize it through the GitHub api, we did some similar thing for gitlab.

**Reply by Davidy22** [reply]
Latest NEWS.rst that I generated is [here](https://raw.githubusercontent.com/Guake/guake/master/NEWS.rst) just need to move a few lines to get differences from 3.7 to 3.8. I'll probably be just copying the notes into the github release like I did with the first rc, also considering just switching to one of those fancy commit message changelog tools and deprecating our hefty chunk of release note files.

But even if we did have auto release note to github releases, I can't really do the release because I don't have the keys to pypi and that's supposed to be something we're supporting.

Also not completely connected to releases, but the readthedocs page is some versions behind the repo and I'm not sure it's pointing at this repo anymore. Made an issue at #1925, do you know who has access to it?

**Reply by gsemet** [reply]
ok i see readthedoc if there is an issue

thanks for the news.rst, sadly i see the rc are not well handled (they should be ignored actually). that's another thing to do later.. i do the release
